.github/workflows/tests.yml

```
@@ -0,0 +1,80 @@
+name: JFrog CLI Tests
+on:
+  push:
+    branches:
+      - v2
+      - v1
+      - dev
+      - dev-v1
+    # Triggers the workflow on labeled PR only.
+  pull_request_target:
+    types: [labeled]
+jobs:
+  CLI-Tests:
+    name: ${{ matrix.product }} ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        product: [artifactory, pip, distribution, xray, plugins, npm, maven, gradle, nuget, go]
```

This is not a product. What do you think about `suite`?
```suggestion
        suite: [artifactory, pip, distribution, xray, plugins, npm, maven, gradle, nuget, go]
```